pkg/apis/agones/v1/gameserver.go

```diff
@@ -373,6 +430,16 @@ func (gss *GameServerSpec) Validate(devAddress string) ([]metav1.StatusCause, bo
 		}
 	}
 
+	if !runtime.FeatureEnabled(runtime.FeatureSafeToEvict) {
+		if gss.Eviction.Safe != "" {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueNotSupported,
+				Field:   "safeToEvict",
```

Comment:
I found one other place where I think the field name needs to be updated.

Reply:
Fixed!